addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3608,14 +3612,15 @@ class SnippetsMenu extends Component {
                         // (mutexed as well).
                         dragAndDropResolve();
 
+                        this.__postSnippetDropExtraActions = () => {
+                            // Restore editor to its normal edition state, also
+                            // make sure the undroppable snippets are updated.
+                            this._disableUndroppableSnippets();
+                            this.options.wysiwyg.odooEditor.unbreakableStepUnactive();
+                            this.options.wysiwyg.odooEditor.historyStep();
+                            this.$el.find('.oe_snippet_thumbnail').removeClass('o_we_already_dragging');
+                        };
```

Comment:
This stuff was meant as a hacky way to handle things in stable. In master, this should definitely be passed as a parameter of `callPostSnippetDrop`, instead of through the `this` object and use of `delete` later